esp/platform/txsummary.cpp

```diff
@@ -126,6 +132,11 @@ StringBuffer& CTxSummary::TxEntryTimer::serialize(StringBuffer& buf, const LogLe
         buf.append(fullname);
         buf.appendf("=%" I64F "u;", value->getTotalMillis());
     }
+    else if (requestedStyle & TXSUMMARY_FWD_OTEL)
+    {
+        if (otelConnector)
+            otelConnector->forwardAttribute(name, nullptr, value->getTotalMillis(), TxUnits::millis, queryLogLevel(), queryGroup());
```

Comment:
forwardAttribute implies some sort of connection, what is that function actually doing? Normalizing? Transforming?

Reply:
In this case, the current cumulative timer value is passed to the current thread's connector. The connector will normalize the name and scale the milliseconds measurement. This is triggered during ESP context destruction when the root server span should be the active span.